packages/core/src/StaticContainer.tsx

```diff
@@ -8,6 +8,10 @@ function StaticContainer(props: any) {
 }
 
 export default React.memo(StaticContainer, (prevProps: any, nextProps: any) => {
+  if (Object.keys(prevProps).length !== Object.keys(nextProps).length) {
+    return false;
+  }
+
```

Comment:
if we're going through the trouble of creating arrays here, we could ditch the code below in favour of a reduce method since we're looping anyway:
```
const prevKeys = Object.keys(prevProps);
const nextKeys = Object.keys(nextProps);
return prevKeys.length === nextKeys.length
  ? prevKeys.reduce((returnValue, key) => returnValue ? (prevProps[key] === nextProps[key]) : false, true)
  : false;
```

Reply:
I'm not sure reduce is a good idea this case as it would extend the iterations beyond the required number of them. But I did reuse the keys extracted from props instead of getting them again.